pandas/core/generic.py

```diff
@@ -6648,7 +6648,12 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
             * 'outside': Only fill NaNs outside valid values (extrapolate).
 
             .. versionadded:: 0.21.0
+        maxgap : int, optional
```

Comment:
Probably `max_gap`.

Reply:
Took the name from jreback's suggestion [here](https://github.com/pandas-dev/pandas/issues/12187#issuecomment-221379702). If I am free to chose I would also go with `max_gap`.

Reply:
yep ``max_gap`` is good